spec/spec_helper.lua

```diff
@@ -1,5 +1,47 @@
-require 'ngx_helper'
+do
+  local function getlocal(fn, var)
+    local i = 1
+    while true do
+      local name, val = debug.getlocal(fn, i)
+      if name == var then
+        return val
+      elseif not name then
+        break
+      end
+      i = i + 1
+    end
+  end
+
+  local function getupvalue(fn, var)
+    local i = 1
+    while true do
+      local name, val = debug.getupvalue(fn, i)
+      if name == var then
+        return val
+      elseif not name then
+        break
+      end
+      i = i + 1
+    end
+  end
+
+  local busted = getlocal(6, 'busted')
```

Comment:
6?

Reply:
:-D 6 layers of stack deep, mind blowing :) will write a comment